include/swift/AST/ASTBridging.h

```diff
@@ -85,6 +85,26 @@ struct BridgedLocatedIdentifier {
   BridgedSourceLoc NameLoc;
 };
 
+struct BridgedConsumedLookupResult {
+  SWIFT_NAME("name")
+  BridgedIdentifier Name;
+
+  SWIFT_NAME("nameLoc")
+  BridgedSourceLoc NameLoc;
+  
+  SWIFT_NAME("flag")
+  SwiftInt Flag;
+  
+#ifdef USED_IN_CPP_SOURCE
+  BridgedConsumedLookupResult(swift::Identifier name,
+                              swift::SourceLoc sourceLoc,
+                              SwiftInt flag
+                              ) : Name(BridgedIdentifier(name)),
```

Comment:
Nit: we typically put the ) on the same line and : on the following. @gottesmm can also help you set up clang-format if you'd rather fix stylistic issues automatically

Reply:
Thank you for pointing this out! I manually run clang-format now for this part of the file. Is there maybe a way to automate this process and run with every commit or just run it on my changes?